website/docs/reference/dbt-commands.md

```diff
@@ -22,9 +22,9 @@ You can run dbt commands in your specific tool by prefixing them with `dbt`.  Fo
 | [build](/reference/commands/build) | Build and test all selected resources (models, seeds, snapshots, tests) | All | All [supported versions](/docs/dbt-versions/core) |
 | cancel  | Cancels the most recent invocation.| dbt Cloud CLI | Requires [dbt v1.6 or higher](/docs/dbt-versions/core) |
 | [clean](/reference/commands/clean) | Deletes artifacts present in the dbt project | All | All [supported versions](/docs/dbt-versions/core) |
-| [clone](/reference/commands/clone) | Clone selected models from the specified state | dbt Cloud CLI <br /> dbt Core | Requires [dbt v1.6 or higher](/docs/dbt-versions/core)  |
+| [clone](/reference/commands/clone) | Clone selected models from the specified state | All | Requires [dbt v1.6 or higher](/docs/dbt-versions/core)  |
 | [compile](/reference/commands/compile) | Compiles (but does not run) the models in a project | All | All [supported versions](/docs/dbt-versions/core) |
-| [debug](/reference/commands/debug)  | Debugs dbt connections and projects  | dbt Core | All [supported versions](/docs/dbt-versions/core) |
+| [debug](/reference/commands/debug)  | Debugs dbt connections and projects  | All | All [supported versions](/docs/dbt-versions/core) |
```

Comment:
I actually don't think debug is in the cloud cli yet 😅

Reply:
oh thank you for catching this!